dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -505,9 +505,19 @@
          *
          * By default it is not set, so the feature is turned off.
          */
-        MailOverseerAddress
+        MailOverseerAddress,
+
+        /**
+         * Following two properties if set to 'true' will enable application to show links
+         * to pages with Privacy Policy and Terms of Use in the footer.
+         *
+         * By default these are set to 'false', so links won't be shown.
+         */
```

Comment:
I'd split this comment into to separate pieces each for its respective property since their usage is not, strictly speaking, connected.